### PR TITLE
Simplify controls

### PR DESCRIPTION
**Pull request by ellisrichardj**
This PR is to adapt Auspice to reflect the design changes that were suggested following the A/B testing and development of the wireframes.  

Much of the change is relating to the removal of components that will not be required for the ViewBovis version, resulting in a much cleaner sidebar.  
image

Information panels have also been updated to be more APHA specific, to provide users with better guidance.
image

Finally installation instructions have also been added

### CODE REVIEW

File: APHA_install.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Install argparse (required by Auspice)
+
+`cd auspice && npm install argparse`
+
+Build Auspice
+
+`npm install --global .`
+
+Correct installation of Auspice can be confirmed with `auspice --help`
```
Comment (Hira-Naveed) [inline APHA_install.md:23]: This is super helpful!

File: src/components/controls/controls.js [hunk 2]
```diff
@@ -2,29 +2,31 @@ import React from "react";
...
+//import ChooseSecondTree from "./choose-second-tree";
+//import ChooseMetric from "./choose-metric";
+//import TransmissionLines from './transmission-lines';
+//import NormalizeFrequencies from "./frequency-normalization";
+//import AnimationOptions from "./animation-options";
+//import ToggleTangle from "./toggle-tangle";
+//import MeasurementsOptions from "./measurementsOptions";
 
 function Controls({mapOn, frequenciesOn, measurementsOn, mobileDisplay}) {
   const { t } = useTranslation();
```
Comment (Hira-Naveed) [inline src/components/controls/controls.js:32]: This is really great and makes the sidebar a lot less busy. I just had 2 things i wanted to mention. 
1. The APHA logo: i do not think we will need that as we will have this displaying in the nav bar of the app, we could potentially replace this with a back button for the splash page? 
2. Other tree visualisation options: I believe this was removed in an earlier version of the Nextstrain Dashboard after A/B testing, users especially likes the 'radial view' of the tree for reports. I think maybe for the first round of the ALPHA testing we could include this to understand more on this user need?

File: src/components/controls/miscInfoText.js [hunk 3]
```diff
@@ -3,55 +3,52 @@ import React from "react";
...
 
-export const ExplodeTreeInfo = (
+/*export const ExplodeTreeInfo = (
   <>This functionality is experimental and should be treated with caution!
     <br/>Exploding a tree by trait X means that for each branch where the trait changes value, we will
     prune off the branch and create a separate (sub)tree.
     It works best when the trait doesn&apos;t change value too frequently.
   </>
-);
+);*/
```
Comment (Hira-Naveed) [inline src/components/controls/miscInfoText.js:54]: Great to see ViewBovis tailored comments in the information panels :)

**Review by Hira-Naveed — APPROVED** [review]
This is great Richard, it was super helpful to also see the screen shots and how great the User Interface is coming along!  I just had one comment on the APHA logo and the Tree options (Layout). Thanks!

File: src/components/controls/controls.js (hunk 2 above)
Comment (ellisrichardj) [reply src/components/controls/controls.js:32]: The APHA logo is part of the splash page and clicking on it takes you back to the splash page - so perhaps there's no need for a back button.  For the tree visualizations, my understanding was that it was a single user who liked it?  I think it adds too much confusion for less experienced users

Comment (ellisrichardj) [reply src/components/controls/controls.js:32]: Yes - it goes back to the Nextstrain splash page at the moment, but its possible to point auspice to an alternate splash page

**Pull request merged**